pipeline-modules/app-service/azure/go-scan.yaml

```diff
@@ -0,0 +1,53 @@
+provisioner: azure
+name: go-scan
+version: 1
+revision: 1
+displayName: Go Scan
+description: Scans go code for vulnarabilities using gosec and golint
+target: ""
+keywords:
+  - Go
+  - Scan
+  - lint
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    go_sec_args:
+      title: Go sec args
+      description: Arguments for Go sec command
+      type: array
+      items:
+        type: string
+    go_lint_args:
+      title: Go lint args
+      description: Arguments for Go lint command
+      type: array
+      items:
+        type: string
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+  required:
+    - go_sec_args
+    - go_lint_args
+  internal:
+    - working_dir
+template: |
+  steps:
+  - script: |
+      echo Installing Gosec
+      wget -O - -q https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s v2.9.6
```

Comment:
should we parameterize this version #?

Reply:
although I have experienced some issues while working with older versions, I updated the template to have current versions defaulted  to variables.